src/giddyup.erl

```diff
@@ -27,6 +27,7 @@
 get_suite(Platform) ->
```

Comment:
@brett since we are copying the logs and configs to a local results directory, there is no need to wait.  We simply give the reporter the location of the logs in the results directory.  Also, we will need to this model when we support parallelization since r_t will not be physically co-located with the Riak nodes.